.env.local

```
@@ -61,3 +61,5 @@ PAGINATED_MESSAGES=NO
 MVL_ENABLED=NO
 # CGN new merchants features
 CGN_MERCHANTS_V2_ENABLED=NO
+# Opt-in payments method
+OPT_IN_PAYMENT_METHODS = NO
```

```suggestion
BPD_OPT_IN_PAYMENT_METHODS = NO
```
Since is strictly related to the BPD iniziative, could it be a better naming?

---

Yeah, refactored in [989152f](https://github.com/pagopa/io-app/pull/3697/commits/989152f5822beb64364772fa77b31dd891bb0d87)